.github/scripts/get-version.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -e
 
-grep -Po "val stableVersion = \"\K[0-9]+.[0-9]+.[0-9]+" version.gradle.kts
+grep "val stableVersion = " version.gradle.kts | grep -Eo "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
`grep -P` stopped working for me, seems it's not very standard: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1596#discussion_r1883052063